parser/src/cfg/defcfg.rs

```diff
@@ -381,6 +385,20 @@ pub fn parse_defcfg(expr: &[SExpr]) -> Result<CfgOptions> {
                             cfg.linux_opts.linux_device_detect_mode = detect_mode;
                         }
                     }
+                    "linux-debounce-duration" => {
+                        #[cfg(any(target_os = "linux", target_os = "unknown"))]
+                        {
+                            cfg.linux_opts.linux_debounce_duration_ms =
+                                parse_cfg_val_u16(val, label, false)?;
+                        }
+                    }
+                    "linux-debounce-algorithm" => {
+                        #[cfg(any(target_os = "linux", target_os = "unknown"))]
+                        {
+                            let algorithm = sexpr_to_str_or_err(val, label)?;
+                            cfg.linux_opts.linux_debounce_algorithm = algorithm.to_string();
```

Comment:
There should be validation and errors here to restrict to known subset of valid algorithms.